Update v1.5.x docs with latest changes (#139)
NordicSemiconductor/asset-tracker-cloud-docs: docs/azure/GettingStarted/Deploy.rst

      # add to .envrc
      export APP_NAME="nrfassettracker"

#. Choose a resource group and a name for the Device Update instance and export it as ``ADU_RESOURCE_GROUP`` and ``ADU_INSTANCE_NAME``.

>    There is a limit of `2 instances per subscription <https://docs.microsoft.com/en-us/azure/iot-hub-device-update/device-update-resources#device-update-instance>`_, so the ADU instance needs to be shared between multiple IoT hubs.

Reviewer: ```suggestion
   There is a limit of `two instances per subscription <https://docs.microsoft.com/en-us/azure/iot-hub-device-update/device-update-resources#device-update-instance>`_, so the ADU instance needs to be shared between multiple IoT hubs.
```

>    In this example, we use ``nRFAssetTrackerADU`` as the resource group and Device Update instance name.

Reviewer: ```suggestion
   In this example, ``nRFAssetTrackerADU`` is used as the resource group and Device Update instance name.
```

   .. code-block:: bash

      # add to .envrc
      export ADU_RESOURCE_GROUP="nRFAssetTrackerADU"
      export ADU_INSTANCE_NAME="nRFAssetTrackerADU"

#. Configure your preferred location (you can list the locations using ``az account list-locations``) and export it on the environment variable ``LOCATION``.

>    In this example, we use ``northeurope`` as the location name.

Reviewer: ```suggestion
   In this example, ``northeurope`` is used as the location name.
```
